x-pack/plugins/security_solution/public/overview/pages/data_quality.tsx

```diff
@@ -200,46 +202,50 @@ const DataQualityComponent: React.FC = () => {
     [createCaseFlyout]
   );
 
+  if (isSourcererLoading) {
```

Comment:
```suggestion
  if (isSourcererLoading || isSignalIndexNameLoading) {
```
Why not move both conditions up?